### PR TITLE
[Server] Seed keys for local provider

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Notes for Reviewers**

This PR fixes #

This PR seeds necessary user keys for the local provider, This PR will not be a blocker for other permission-based PRs which I'll be raising for the UI

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
@MUzairS15 This PR is almost ready I guess, however only thing left is where to store keys so that we can read from there and seed them, I didn't push my changes to gitwalker as we are not going to do that.

**Review by MUzairS15 — APPROVED** [review]
@Yashsharma1911 LGTM.
Added logic for seeding keys for local provider.

**Comment by MUzairS15** [comment]
@leecalcote @Yashsharma1911 The Keys spreadsheet needs to be published and the workflow to be updated (only one-time update).

File: .github/workflows/generate_keys.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Generate Keys
```
Comment (MUzairS15) [inline .github/workflows/generate_keys.yml:1]:
```suggestion
name: Import Keys
```

**Comment by MUzairS15** [comment]
Yes it was already. I have also added the keys.csv file.

File: .github/workflows/generate_keys.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    if: github.repository == 'meshery/meshery'
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GH_ACCESS_TOKEN }}
+        fetch-depth: 1
+    - name: Set spreadsheet_uri as environment variable
+      run: echo "spreadsheet_uri=https://docs.google.com/spreadsheets/d/e/2PACX-1vT66uDq7Fan9GIjba7qS8jwpr7hu69P_iF4RR52hD-yVVtm2Pybbi67iq9tfYWwO4KtBPb7OiP4ImQn/pub?gid=829069645&single=true&output=csv" >> $GITHUB_ENV
```
Comment (MUzairS15) [inline .github/workflows/generate_keys.yml:20]:
```suggestion
      run: echo "spreadsheet_uri=" >> $GITHUB_ENV
```

**Comment by MUzairS15** [comment]
Merging after build finishes.

File: server/models/default_local_provider.go [hunk 3]
```diff
@@ -1132,6 +1130,31 @@ func (l *DefaultLocalProvider) SeedContent(log logger.Handler) {
...
+			},
+			{
+				ID:          uuid.FromStringOrNil("9030a03b-70b3-4956-afb4-a6b7935cd478"),
+				Owner:       uuid.Nil,
+				Function:    "Create new design",
+				Category:    "Catalog Management",
+				Description: "Create new Meshery design",
+				Subcategory: "Designs",
+			},
+		}
```
Comment (Yashsharma1911) [inline server/models/default_local_provider.go:1151]: @MUzairS15 we don't require this right?

File: server/models/default_local_provider.go [hunk 4]
```diff
@@ -1132,6 +1130,31 @@ func (l *DefaultLocalProvider) SeedContent(log logger.Handler) {
...
+		_, err := l.KeyPersister.SaveUsersKeys(keys)
+		if err != nil {
+			log.Error(ErrGettingSeededComponents(err, "key"))
+		}
+	}()
```
Comment (Yashsharma1911) [inline server/models/default_local_provider.go:1156]: We can remove this whole I guess

Comment (MUzairS15) [reply server/models/default_local_provider.go:1156]: yes

Comment (Yashsharma1911) [reply server/models/default_local_provider.go:1156]: Thank you I'll remove and merge

**Comment by Yashsharma1911** [comment]
@MUzairS15 thankyou

**Pull request merged**